packaging/opensuse/snapd.spec

```diff
@@ -134,13 +134,22 @@ the system:snappy repository.
 # Generate autotools build system files
 cd cmd && autoreconf -i -f
 
-# Enable hardening; We can't use -pie here as this conflicts with
-# our build of static binaries for snap-confine. Also see
-# https://bugzilla.redhat.com/show_bug.cgi?id=1343892
+# Enable hardening; Also see https://bugzilla.redhat.com/show_bug.cgi?id=1343892
 CFLAGS="$RPM_OPT_FLAGS -fPIC -Wl,-z,relro -Wl,-z,now"
 CXXFLAGS="$RPM_OPT_FLAGS -fPIC -Wl,-z,relro -Wl,-z,now"
+LDFLAGS=""
+# On openSUSE Leap 15 or more recent build position independent executables.
+# For a helpful guide about the versions and macros used below, please see:
+# https://en.opensuse.org/openSUSE:Build_Service_cross_distribution_howto
+%if (0%{?sle_version} >= 150000 && 0%{?is_opensuse}) || %if 0%{?suse_version} > 1500
+CFLAGS="$CFLAGS -fPIE -fpie"
+CXXFLAGS"$CXXFLAGS -fPIE -pie"
```

Comment:
You don't set both `-fPIE` and `-fpie`. Only set `-fPIE`.

Reply:
Thanks, I'll tweak that,